sdk/extensions/Microsoft.Extensions.Azure/tests/ClientFactoryTests.cs

```diff
@@ -338,18 +344,27 @@ public void CreatesDefaultAzureCredential(
 
             // TODO: Since these can't build with project reference, we have to comment them out for now.
             // When we resolve https://github.com/Azure/azure-sdk-for-net/issues/45806, we can add them back.
```

Comment:
If we're ready to uncomment this, should we change the comment?

Reply:
thanks - forgot to delete the comment!